controllers/operatorpipeline_controller.go

```diff
@@ -97,5 +209,8 @@ func (r *OperatorPipelineReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&imagev1.ImageStream{}).
 		Owns(&tekton.Pipeline{}).
 		Owns(&tekton.Task{}).
+		Owns(&securityv1.SecurityContextConstraints{}).
+		Owns(&rbacv1.ClusterRole{}).
+		Owns(&rbacv1.ClusterRoleBinding{}).
```

Comment:
Hrm, do you still need to own these resources? Or do you need `For` instead?

Reply:
This `owns` is inside a `for` for `OperatorPipeline`, and we are adding an owner to each one of these resources, maybe this diff just doesn't show this properly, but I think this is already how you expect it. Please lmk if you are thinking something else.

Reply:
Did you end up getting events when a user deletes the CRB/CR/SCC that's owned by this controller? To remake it?

Reply:
The reconciler does not get events for the cluster scoped resources, so if they are deleted on accident by a user we can't recreate them.

Reply:
Just closing the loop - you _can_ recreate them, but you probably don't automatically recreate them on deletion. Just when you go back through your reconcile loop. Neither here nor there, resolving this comment thread.